ext/bcmath/tests/bcadd_large_numbers.phpt

```diff
@@ -0,0 +1,276 @@
+--TEST--
+bcadd() function with large numbers
+--EXTENSIONS--
+bcmath
+--INI--
+bcmath.scale=0
+--FILE--
+<?php
+require(__DIR__."/run_bcmath_tests_function.inc");
+
+$leftSummands = [
+    "15151324141414.412312232141241",
+    "-132132245132134.1515123765412",
+    "141241241241241248267654747412",
+    "-149143276547656984948124912",
+    "0.1322135476547459213732911312",
+    "-0.123912932193769965476541321",
+];
+$rightSummands = array_merge($leftSummands, [
+    "0",
+    "0.00",
+    "-0",
+    "-0.00",
```

Comment:
We're already testing 0 in another test

Reply:
We test it, but as a first arguments of the function. In this test we pass zeroes only as second argument of the function. The same with other tests. Do you think it's too much coverage?

Reply:
Ah right, should be fine then.